lib/lita/handler/common.rb

```diff
@@ -72,10 +72,15 @@ def log_error(robot, error)
       # @return [Robot]
       attr_reader :robot
 
+      # The current unpersisted and shared {State} instance.
+      # @return [State]
+      attr_reader :state
```

Comment:
Don't store or expose the state object directly. Initialize the value for the handler with `robot.state.set(self.class.namespace, {})` and add an instance method called `state` that does `robot.state.get(self.class.namespace)`.